bundles/FileExplorerBundle/src/Controller/Admin/AdminController.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+namespace Pimcore\Bundle\FileExplorerBundle\Controller\Admin;
+
+use Pimcore\File;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/file-explorer")
+ *
+ * @internal
+ */
+class AdminController extends \Pimcore\Bundle\AdminBundle\Controller\AdminController
+{
+    /**
+     * @Route("/tree", name="pimcore_file_explorer_tree", methods={"GET"})
```

Comment:
Change the route names accordingly e.g. pimcore_bundle_glossary_settings_glossary. See [SettingsController](https://github.com/pimcore/pimcore/blob/4d3b5dd592472b9e02fa1fec4ddb09d81b3e9afb/bundles/GlossaryBundle/src/Controller/SettingsController.php)